io/http/server/endpoint listener/server error checker/0faadab5-12bc-4bd8-b7e1-0bd8ddf21a43.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "io/http/server/Endpoint listener/Server error checker",
+  "interface": {
+    "inputs": {
+      "8978272a-83da-48fd-b253-ca7585916edc": {
+        "type": "typeof `unsplit` of `splitter`",
+        "name": "unsplit",
+        "description": "[Inherited from port `unsplit` of `splitter`] \nReceives records or tuples to be split into individual items.\n\nExamples:\n* {\"a\": 5, \"b\": 2}\n* [5, 2]"
+      }
+    },
+    "outputs": {
+      "e7b7f2c0-4e3f-4630-83d6-f5c18f170947": {
+        "type": "typeof `on true` of `fork`",
+        "name": "on true",
+        "description": "[Inherited from port `on true` of `fork`] \nSends signal received via `data` when condition was true."
+      },
+      "bfb24c70-24af-457b-bc18-d945b3f65b3a": {
+        "type": "typeof `on false` of `fork`",
+        "name": "on false",
+        "description": "[Inherited from port `on false` of `fork`] \nSends signal received via `data` when condition was false."
+      }
+    }
+  },
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "d6d7c6a5-d2c3-44f7-a132-61b0de37192b": {
+          "coordinates": [
+            -121,
+            37
+          ],
+          "name": "splitter",
+          "prototype": "73303290-6d1e-44cf-9414-66769cbb8e91",
+          "params": {
+            "7b39d568-3d5a-46fe-8670-3355f60c2746": "[\"error\", \"origin\"]"
+          }
+        },
+        "d365c1e4-9785-4817-8313-dc81c63a2240": {
+          "coordinates": [
+            45,
+            106
+          ],
+          "name": "is application error",
```

Comment:
What is "application" error? Server error?

Reply:
rewrote to server.